packages/client/components/PokerEstimateHeaderCard.tsx

```diff
@@ -13,6 +16,81 @@ import PokerEstimateHeaderCardContent, {
 import PokerEstimateHeaderCardError from './PokerEstimateHeaderCardError'
 import PokerEstimateHeaderCardParabol from './PokerEstimateHeaderCardParabol'
 
+const refreshStoryIntegrationQuery = graphql`
+  query PokerEstimateHeaderCardQuery($meetingId: ID!, $storyId: ID!) {
+    viewer {
+      meeting(meetingId: $meetingId) {
+        ... on PokerMeeting {
+          story(storyId: $storyId) {
```

Comment:
we can break this bit out into a fragment & reuse it between this query & the original fragment. that way if we update the fragment we don't have to remember to update this, too